source/game/field/ObjectFlowTable.hh

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Common.hh>
+
+namespace Field {
+
+struct SObjectCollisionSet {
```

Comment:
Add `/// @brief` to describe the purpose of this struct. Also may be worthwhile to add descriptions for what `mode` and `resources` are, because I certainly don't know.

Reply:
If we don't have a great way to explain this struct, can you leave a `/// @todo` comment?